multiform/forms.py

```diff
@@ -8,7 +8,12 @@
 
 from django.core.exceptions import ImproperlyConfigured
 from django.forms.forms import BaseForm
-from django.forms.util import ErrorList
+
+try:
+    from django.forms.util import ErrorList
```

Comment:
Could you first try to import the `utils` package and fall back to `util`, please. This is the way we handle it in Django itself.